files-override/ts/tests/test-helper.ts

```diff
@@ -0,0 +1,14 @@
+import Application from '<%= name %>/app';
+import config from '<%= name %>/config/environment';
+import * as QUnit from 'qunit';
+import { setApplication } from '@ember/test-helpers';
+import { setup } from 'qunit-dom';
+import { start as qunitStart } from 'ember-qunit';
+
+export function start() {
+  setApplication(Application.create(config.APP));
+
+  setup(QUnit.assert);
+
+  qunitStart();
+}
```

Comment:
what's different about this file? 🤔 if I'm not mistaken this is exactly the same as the JS file, no? why do we need to produce this file if it's exactly the same?

Reply:
the file extension -- we need to override the upstream blueprint's file, be than js or ts, and we can't have both, else the build errors.
(another case for just splitting from the upstream blueprint)

Reply:
Sorry I don't understand 🤔 the only place that this is being imported is in https://github.com/embroider-build/app-blueprint/blob/main/files-override/js/tests/index.html#L33 and that is without the extension

Reply:
without this file being here, we'll get the test-helper.ts from the upstream blueprint